packaging/windows/registrykeys.wxs

```diff
@@ -25,7 +25,6 @@
         </RegistryKey>
 
         <Environment Id="E_PATH" Name="PATH" Value="[DOTNETHOME]bin" Part="last" Action="set" System="yes" />
-        <Environment Id="E_DOTNET_HOME" Name="DOTNET_HOME" Value="[DOTNETHOME]" Part="all" Action="set" System="yes" />
```

Comment:
Looks fine. Now'ish would be the best time to maybe change the rest of the installation to maybe use a more standard override for installing, like: `INSTALLFOLDER` or `INSTALLDIR` (whatever you prefer). Right now it is `DOTNETHOME` which doesn't even match the environment variable name. :smile_cat: 

Up to you.